tools/ci_build/github/azure-pipelines/templates/test_linux.yml

```diff
@@ -0,0 +1,60 @@
+parameters:
```

Comment:
What's the purpose of this file? Is it possible to write a line or two at the start of yml files going forward (where not obvious) to make this clear? Same comment for others.

Reply:
Test nuget package.